template/src/services/networking/index.ts

```diff
@@ -0,0 +1,87 @@
+import {AxiosError, AxiosRequestHeaders, AxiosResponse} from 'axios'
+import {IAxiosMethod} from '../../constants/interface/services/axios'
+import {isObject} from '../../utilities/utils'
+import instance from './axios'
+
+const AxiosMethod: IAxiosMethod = {
+  get: 'GET',
+  post: 'POST',
+  put: 'PUT',
+  delete: 'DELETE',
+  patch: 'PATCH',
+}
+
+async function axiosAPI<T>({
+  url,
+  method,
+  data,
+  config,
+}: {
+  url: string
+  method: string
+  data?: T | string
+  config?: AxiosRequestHeaders
+}): Promise<AxiosResponse<T>> {
```

Comment:
I think we should separate Type to another place:
Ex: 
type Param<T> = {
 url: string, 
 ...
}
=> async function axiosAPI<T>(params: Param<T>): Promise<AxiosResponse<T>>

Reply:
done